vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -83,7 +104,7 @@ To ensure a deterministic answer to our queries, we've set the temperature to 0.
 
 Passing an instantiated model to `VizroAI` lets you customize it, and additionally, it enables you to use an OpenAI model that is not included in the above list of [supported models](#supported-models).
 
-## Azure OpenAI models
+### Azure OpenAI models
```

Comment:
The Azure issue is not fully resolved yet. Shall we drop this section for now? @maxschulz-COL

Reply:
Yes please, let's take it out for now @lingyielia